config.sample.yaml

```diff
@@ -241,3 +241,9 @@ ircService:
   # dump .db files to. This is relative to the project directory.
   # Required.
   databaseUri: "nedb://data"
+  
+  # Optional. The IPv6 prefix to use for generating unique addresses for each
+  # connected user. If not specified, all users will connect from the same
+  # (default) address.
+  uniqueIPv6:
```

Comment:
This is *not* assigning unique IPv6 addresses for each *Matrix* connection, so it doesn't make much sense being here. Please move this to the `ircClients` section.